src/en/news/blog/2023/v18.2.0-released/index.md

```diff
@@ -0,0 +1,523 @@
+---
+title: "v18.2.0 Reef released"
+date: "2023-08-07"
+author: "Laura Flores"
+tags:
+  - "release"
+  - "reef"
+---
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid (Sepioteuthis).
+
+This is the first stable release of Ceph Reef.
+
+Contents:
+
+- [Major Changes from Quincy](#changes)
+- [Upgrading from Pacific or Quincy](#upgrade)
+- [Upgrading from pre-Pacific releases (like Octopus)](#upgrade-from-older-release)
+- [Thank You to Our Contributors](#contributors)
+
+## <a id="changes"></a>Major Changes from Quincy
+
+### Highlights
+
+See the relevant sections below for more details on these changes.
+
+- **RADOS:** RocksDB has been upgraded to version 7.9.2.
+
+- **RADOS:** The ``perf dump`` and ``perf schema`` commands have been deprecated in favor of the new ``counter dump`` and ``counter schema`` commands.
+
+- **RADOS:** Cache tiering is now deprecated.
+
+- **RADOS:** A new feature, the "read balancer", is now available, which allows users to balance primary PGs per pool on their cluster.
```

Comment:
nit: _which allows users to balance primary PGs per pool on their cluster_. Shouldn't be _clusters_?

Reply:
Fixed